components/script/dom/urlsearchparams.rs

```diff
@@ -168,6 +168,15 @@ impl URLSearchParamsMethods for URLSearchParams {
         self.update_steps();
     }
 
+    // https://url.spec.whatwg.org/#dom-urlsearchparams-sort
+    fn Sort(&self) {
+        // Step 1.
+        self.list.borrow_mut().sort_by(|(a, _), (b, _)| a.cmp(&b));
```

Comment:
With this implementation, we only compare with code points but, per the spec, it said `Sorting must be done by comparison of code units.`.

But here comes a confused point for me.

What's the priority between code points and code units?
If code points is higher, maybe the result of test case `ﬃ&🌈` is incorrect now? because `ﬃ` has bigger code points so that we might not need to compare code units?

If code units is higher, maybe the result of following test case is incorrect? because `aaa` has `3` code units, it should be bigger than `bb`?

https://github.com/web-platform-tests/wpt/blob/1d3a454b4af3f3d70206b53a6dce49ae0ddcc7a9/url/urlsearchparams-sort.any.js#L22-L25

Reply:
Thanks! I'm totally confused by the "code unit" here 😂 I'll try `.encode_utf16()`!